SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -235,6 +253,13 @@ private static int CreateSimDeployments(string[] args, bool useChinaPlatform)
                 return 1;
             }
 
+            var autoConnect = false;
+            if (!Boolean.TryParse(args[10], out autoConnect))
```

Comment:
```suggestion
            if (!Boolean.TryParse(args[11], out autoConnect))
```

Reply:
You also need to update the usage and bump the number of args for "createsim" in `Main`.